src/accordion/types.ts

```diff
@@ -8,6 +8,7 @@ export interface AccordionProps {
     "data-testid"?: string | undefined;
     className?: string | undefined;
     headingLevel?: number | undefined;
+    expandCollapseClickCallback?: () => void,
```

Comment:
rename callback to match react naming conventions, add `| undefined` to support [exactOptionalPropertyTypes](https://www.typescriptlang.org/tsconfig/exactOptionalPropertyTypes.html)

also propose to provide the next state of the accordion

```suggestion
    onExpandCollapseChange?: ((expanded: boolean) => void) | undefined,
```